kolibri/content/views.py

```diff
@@ -54,8 +51,8 @@ def get(self, request, zipped_filename, embedded_filepath):
             else:
                 # load the stream from json file into memory, replace the path_place_holder.
                 content = zf.open(info).read()
-                str_to_be_replaced = str("\\n\\n![](/" + request.resolver_match.url_name + "/" + zipped_filename)
-                content_with_path = content.replace(path_place_holder, str_to_be_replaced)
+                str_to_be_replaced = str('/' + request.resolver_match.url_name + "/" + zipped_filename)
+                content_with_path = content.replace('$' + exercises.IMG_PLACEHOLDER, str_to_be_replaced)
```

Comment:
yes, you need to run `pip install le_utils==0.0.9rc7`, maybe also need to unintall le_utils==0.0.8 before instal 0.0.9rc7